book/website/project-design/project-management-methodologies/overview.md

```diff
@@ -0,0 +1,30 @@
+# Project Management Methodologies/Frameworks Overview
+
+Choosing the right project management framework for a research project is crucial, as it can significantly influence your research project's success, efficiency, and potential for collaboration.
+Research projects are often characterised by uncertainty, evolving objectives, and a need for flexibility.
+As such, they require a framework that can accommodate these dynamics.
+As a recent study pointed out, "a fine balance is needed between implementing management approaches to support the efficient and effective delivery of projects while not compromising accountability, creativity and innovation." [santos2022framework]
+
+Traditional methodologies, for example Waterfall, offer a structured, linear approach beneficial for projects with well-defined stages and stable requirements, known from the beginning of the project design process.
+
+Agile frameworks, including Scrum and Kanban, are flexible and can adapt to changing research landscapes and iterative discovery processes that are part and parcel of many research projects.
```

Comment:
```suggestion
Agile frameworks, including Scrum and Kanban, are flexible and can adapt to changing research landscapes and iterative discovery processes that are part of many research projects.
```

Nothing wrong with the phrasing but it's an English idiom which might be tricky to translate in the future

Reply:
Makes sense!